lib/steps.js

```diff
@@ -78,8 +79,9 @@ Given(/I start the interview at "([^"]+)"[ on ]?(.*)/, async (INTERVIEW_URL, opt
 
   scope.activate = click_with[ scope.device ];
 
+  let interview_url = `${ base_url }${ file_name }.yml`;
```

Comment:
Should this be a `const`?

Reply:
The use of `const` vs. `let` is a whole philosophical topic of its own, but it's legitimate in both ways as far as javascript is concerned. Shall we discuss it separately from this PR?